factories/gift.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require "faker"
+
+FactoryGirl.define do
```

Comment:
Since this is a new factory, it should probably have a [model test](https://github.com/otwcode/otwarchive/blob/master/spec/models/gift_spec.rb) along the lines of

```
  it "has a valid factory" do
    expect(build(:gift)).to be_valid
  end
```